### ISSUE TITLE
missing icon pack on cinnamon theme

### ISSUE THREAD

**Issue by vincent456** [B00merang-Project/Windows-Vista#3]
hello.
The icons for windows vista are missing from the cinnamon theme, can't even find them at https://b00merang.weebly.com/icon-themes.html

**Comment by Elbullazul** [comment]
yes, we didn't think making one was necessary because of the similarity with Windows 7's icon theme
